buildSrc/src/main/kotlin/io/spine/internal/gradle/Repositories.kt

```diff
@@ -170,15 +172,49 @@ object Repos {
 }
 
 /**
- * The function to be used in `buildscript` clauses when fully-qualified call must be made.
+ * Registers the standard set of Maven repositories.
+ *
+ * To be used in `buildscript` clauses when fully-qualified call must be made.
  */
 @Suppress("unused")
 fun doApplyStandard(repositories: RepositoryHandler) {
     repositories.applyStandard()
 }
 
+/**
+ * Registers the selected GitHub Packages repos as Maven repositories.
+ *
+ * To be used in `buildscript` clauses when fully-qualified call must be made.
+ *
+ * @see applyGitHubPackages
+ */
+@Suppress("unused")
+fun doApplyGitHubPackages(repositories: RepositoryHandler, project: Project) {
+    repositories.applyGitHubPackages(project)
+}
+
+/**
+ * Applies the repositories hosted at GitHub Packages, to which Spine artifacts were published.
+ *
+ * This method should be used by those wishing to have Spine artifacts published
+ * to GitHub Packages as dependencies.
+ */
+fun RepositoryHandler.applyGitHubPackages(project: Project) {
+    val baseTypes = gitHub("base-types")
```

Comment:
Let's pass the repository name as an argument to this `fun`. I suspect, more repos will switch to GitHub Packages in the near future.

Reply:
As discussed, this method will contain more repos from GitHub Packages. Therefore, we won't pass the name to it.